### PR TITLE
Fix  --csv-overrides flag

### PR DESCRIPTION
**Pull request by erkanerol**
Signed-off-by: Erkan Erol <eerol@redhat.com>

While refactoring the codes for Sonar, we broke this part. Here is the fix. 

**Release note**:
```release-note
NONE
```

### CODE REVIEW

**Review by tiraboschi — COMMENTED** [review]
Can we please also add some kind of test/check in hack/build-manifests.sh to prevent future regressions?

**Review by tiraboschi — APPROVED** [review]

File: hack/build-manifests.sh [hunk 1]
```diff
@@ -390,6 +383,11 @@ ${PROJECT_ROOT}/tools/csv-merger/csv-merger \
   --operator-image-name="${HCO_OPERATOR_IMAGE}" \
   --webhook-image-name="${HCO_WEBHOOK_IMAGE}" > "${CSV_DIR}/${OPERATOR_NAME}.v${CSV_VERSION}.${CSV_EXT}"
 
+rendered_csv="$(cat "${CSV_DIR}/${OPERATOR_NAME}.v${CSV_VERSION}.${CSV_EXT}")"
+rendered_keywords="$(echo "$rendered_csv" |grep 'keywords' -A 3)"
```
Comment (orenc1) [inline hack/build-manifests.sh:387]: I think `-A 3` argument on grep is not robust enough, as it will break if the number of keywords will be changed.

**Pull request merged**